rotkehlchen/tasks/assets.py

```diff
@@ -244,3 +260,74 @@ def update_owned_assets(user_db: DBHandler) -> None:
             'INSERT OR REPLACE INTO key_value_cache (name, value) VALUES (?, ?)',
             (DBCacheStatic.LAST_OWNED_ASSETS_UPDATE.value, str(ts_now())),
         )
+
+
+def update_underlying_assets(chains_aggregator: 'ChainsAggregator') -> None:
+    """Fetch the underlying defi assets and polulate `underlying_tokens_list` in globaldb"""
+    for chain_id, data_provider_address in (
+        (ChainID.ETHEREUM, DATA_PROVIDER_ETH),
+        (ChainID.OPTIMISM, DATA_PROVIDER_EVM),
+        (ChainID.POLYGON_POS, DATA_PROVIDER_EVM),
+        (ChainID.ARBITRUM_ONE, DATA_PROVIDER_EVM),
+        (ChainID.BASE, DATA_PROVIDER_BASE),
+        (ChainID.GNOSIS, DATA_PROVIDER_GNO),
+        (ChainID.SCROLL, DATA_PROVIDER_SCRL),
+    ):
+        node_inquirer = chains_aggregator.get_evm_manager(chain_id=chain_id).node_inquirer  # type: ignore[arg-type]  # all iterated ChainIDs are supported
+        aave_data_provider = node_inquirer.contracts.contract(address=data_provider_address)
+
+        try:
+            underlying_tokens = [  # get all the underlying tokens in aave v3
+                to_checksum_address(token_address)
```

Comment:
aren't the addresses already checksummed when they come here? Use string_to_evm_address if possible since `to_checksum_address` is expensive